app/javascript/src/Logger.js

```diff
@@ -0,0 +1,34 @@
+import { Logtail } from '@logtail/browser';
+
+export default class Logger {
+  #environment;
+  #gameId;
+
+  #logtail;
+
+  constructor(environment, gameId) {
+    this.#environment = environment;
+    this.#gameId = gameId;
+
+    this.#logtail = new Logtail('3bdHcA8P3mcww2ojgC5G8YiT');
+  }
+
+  error(message, data) {
+    data.gameId = this.#gameId;
+
+    switch (this.#environment) {
```

Comment:
I wonder if we can use a word other than "environment". I think of "environment" as just being "development", "test", and "production". But if we have a "replay" one then that seems like a different paradigm. It's still in `production`. What if we call it a "scope"? I'd want to use "mode" but we already have game mode elsewhere, I think. What do you think?

Reply:
Yeah I wasn't too keen on having `replay` or even `unittest` as an environment, and didn't have time to think up of a better design

Let's go with the rename of `scope`, I think it quite accurately describes the purpose of this without having to redesign the whole approach